buf.yaml

```diff
@@ -0,0 +1,16 @@
+build:
+  roots:
+    - .
+  excludes:
+    - api/buckets/pb/javascript
+    - api/hub/pb/javascript
+    - api/users/pb/javascript
+lint:
+  use:
+    - DEFAULT
+  except:
+    - PACKAGE_VERSION_SUFFIX
```

Comment:
I'm ignoring this rule for now, but it's actually cool if we want to use it one day. It enforces that your proto package path include a version folder like `v0` so that, when you choose, you can break API compatibility... a client using `some/path/v0/myservice.proto` simply can't talk to a server serving `some/path/v1/myservice.proto`.